src/lib/bidding-calculator/Mainchain.ts

```diff
@@ -145,7 +145,7 @@ class Mainchain {
   public async fetchExchangeRates() {
     const client = await this.client;
     const priceIndex = (await client.query.priceIndex.current()).value;
-    const USD = convertFixedU128ToBigNumber(priceIndex.argonUsdTargetPrice.toBigInt()).toNumber();  
+    const USD = convertFixedU128ToBigNumber(priceIndex.argonUsdPrice.toBigInt()).toNumber();
```

Comment:
Does argonUsdPrice always have a price now? I think it was null on testnet so I reverted to argonUsdTargetPrice for testing purposes. I could be getting this confused with another API call though.

Reply:
I think it should. But I don't think we have any scenario where there's a target price but not a usd price. They're either both set or neither